src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -310,3 +310,17 @@ void MatterGeneralCommissioningPluginServerInitCallback()
     registerAttributeAccessOverride(&gAttrAccess);
     DeviceLayer::PlatformMgrImpl().AddEventHandler(OnPlatformEventHandler);
 }
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace GeneralCommissioning {
+void SetBreadcrumb(Attributes::Breadcrumb::TypeInfo::Type breadcrumb)
+{
+    VerifyOrReturn(DeviceLayer::DeviceControlServer::DeviceControlSvr().GetFailSafeContext().IsFailSafeArmed());
```

Comment:
That does not match the spec, as far as I can tell...

That said, it _might_ be good to enforce this, but needs spec changes, right?

Reply:
Removed